kong-2.8.0-0.rockspec

```diff
@@ -71,6 +71,8 @@ build = {
     ["kong.clustering.utils"] = "kong/clustering/utils.lua",
     ["kong.clustering.compat.removed_fields"] = "kong/clustering/compat/removed_fields.lua",
     ["kong.clustering.config_helper"] = "kong/clustering/config_helper.lua",
+    ["kong.clustering.services.negotiation"] = "kong/clustering/services/negotiation.lua",
+    ["kong.clustering.services.supported"] = "kong/clustering/services/supported.lua",
 
```

Comment:
negotiation.proto should also be added somewhere in the rockspec

Reply:
We don't do this in the rockspec. We will add it to packing in a build script.